core/src/App.html

```diff
@@ -7,14 +7,30 @@
 <script type="text/javascript">
     import TopNav from './TopNav.html';
     import LeftNav from './LeftNav.html';
-    import { handleRouteChange, matchPath, concatenatePath, getNodePath, navigateTo } from './services/routing.js';
+    import { removeInactiveIframes, getActiveIframe, setPreviousActiveIframe, handleRouteChange, matchPath, concatenatePath, getNodePath, navigateTo } from './services/routing.js';
```

Comment:
IMHO code would be easier to understand if we import everything under "Routing" and we call its methods below as Routing.xxx. What do you think?

Reply:
Indeed more descriptive. Refactored.